### ISSUE TITLE
@wordpress/dependency-group Wrong grouping

### ISSUE THREAD

**Issue by bfintal** [WordPress/gutenberg#17050]
**Describe the bug**
I have this sample import statements:

```js
import { format } from '@wordpress/date'
import { pickBy } from 'lodash'
import { PanelBody } from '@wordpress/components'
```

Running `wp-scripts lint-js --fix` on this groups together the dependencies, but doesn't rearrange the location so I end up with:

```js
/**
 * WordPress dependencies
 */
import { format } from '@wordpress/date'

/**
 * External dependencies
 */
import { pickBy } from 'lodash'
import { PanelBody } from '@wordpress/components'
```

You'll have to manually move the `@wordpress/*` dependencies together.

**Expected behavior**
The `@wordpress/*` / External / Internal dependencies would be sorted and grouped together.

**Desktop (please complete the following information):**
Relevant part of my devDependencies:

```json
"devDependencies": {
    "@wordpress/eslint-plugin": "^2.4.0",
    "eslint": "^6.1.0"
}
```

**Comment by swissspidy** [comment]
I've definitely noticed that too and it can be a bit annoying. Would be nice if it could be improved.

**Comment by kathrynwp** [comment]
@gziolo Hi Greg, would you mind taking a peek on this and confirming whether it's still an issue and if so, whether it's on the radar to be looked at? I've marked it as stale in the meantime. Thanks!

**Comment by gziolo** [comment]
Yes, this is still a valid issue. It's a low priority and nice to have, though. The linter is used primarily to catch potential bugs and coding style violations, and automatic fixing is the cherry on the cake.

**Comment by kathrynwp** [comment]
@gziolo Great, thanks for chiming in. I've adjusted the labels accordingly.
